xdebug/xdebug.ini

```diff
@@ -1,9 +1,15 @@
-zend_extension=xdebug
+; The below xdebug settings are part of the esmero-php-debug container. You may override any of the settings or add additional ones (see https://xdebug.org/docs/all_settings) as needed.
+
+;xdebug.remote_handler=dbgp
+;xdebug.client_port=9999
+;xdebug.start_with_request=yes
+;xdebug.connect_timeout_ms=500
+;xdebug.mode=develop,debug
+;xdebug.start_with_request=trigger
```

Comment:
This might be the issue? xdebug.start_with_request twice, last one with "trigger" ?